### ISSUE TITLE
Virtual scroll has a scroll limit (and it's not that large)

### ISSUE THREAD

**Issue by ernestbofill** [angular/material2#14990]
#### What is the expected behavior?
The scroll is next to unlimited in its length

#### What is the current behavior?
Scrolling reaches it's limit at `3.3553e7px`

#### What are the steps to reproduce?
https://stackblitz.com/edit/angular-material2-issue-mlhd9q

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
```json
    "@angular/cdk": "7.0.2",
    "@angular/core": "7.0.1",
```

Google Chrome
Version 71.0.3578.98 (Official Build) (64-bit)

#### Is there anything else we should know?
It doesn't seem to be a browser limit. I have been able to create a larger scroll area than the limit above and also successfully used `translateY()` with a greater value

**Comment by mmalerba** [comment]
I've marked this as P5 because I'm not sure if it is feasible due to the incorrect measurements the browser reports

**Comment by ernestbofill** [comment]
I find it strange that the issue reproduces in any browser that I tested. Is this a known issue?

**Comment by mmalerba** [comment]
Yes, browsers have limits on the size of elements. It would be possible to work around those limits by stacking multiple elements, except that the limit also appears to apply to scrollHeight/scrollTop https://jsfiddle.net/malerba89/gbzwrcuv/

**Comment by mxswat** [comment]
Test on Angular 7 and the issue is still here.
With the latest virtual scroll, the limit is 671088
https://stackblitz.com/angular/pljjovegmex set length to 1000000

**Comment by manju-reddys** [comment]
I wrote blog and implemented a solution to this issue: [Read here](https://medium.com/@manju_reddys/rendering-array-of-billion-of-records-at-60-f-s-in-angular-or-vanilla-js-2613e5983a10)

**Comment by kevhof** [comment]
Are you planning to address this issue soon given that a solution was provided? It's been 3 years now.

**Comment by wagnermaciel** [comment]
@kevhof Hi, this is a very low priority issue and we are a small team so we haven't been able to dedicate any time to looking into this
